src/Containers.jsx

```diff
@@ -59,7 +59,6 @@ const ContainerActions = ({ container, healthcheck, onAddNotification, localImag
                         .catch(ex => {
                             const error = cockpit.format(_("Failed to force remove container $0"), container.Name); // not-covered: OS error
                             onAddNotification({ type: 'danger', error, errorDetail: ex.message });
-                            throw ex;
```

Comment:
This needs some justification/explanation. It looks like the caller *wants* to get failures as promise rejection as well?

Reply:
I checked more carefully: `deleteContainer` is directly wired to an `onClick` handler, so this doesn't expect a failure.